packages/epo-widget-lib/src/atomic/ElapsedTime/ElapsedTime.tsx

```diff
@@ -30,7 +35,7 @@ const ElapsedTime: FunctionComponent<TimeStepProps> = ({
   }, "PT");
 
   return (
-    <Styled.ElapsedTimeContainer>
+    <Styled.ElapsedTimeContainer className={className}>
```

Comment:
added classnames to a bunch of components so they can be re-styled by styled-components